js/app/delivery/AddressBook.js

```diff
@@ -393,6 +413,7 @@ export default function(el, options) {
 
   render(
     <AddressBook
+      allowSearchSavedAddresses={ allowSearchSavedAddresses }
```

Comment:
maybe better to explicitly convert it to boolean like 

`allowSearchSavedAddresses={ Boolean(allowSearchSavedAddresses) }` ?

I suppose it will be `undefined` in all places where it is not set